plugins/jobs/src/js/data/JobModel.ts

```diff
@@ -160,31 +83,14 @@ export const resolvers = ({
       _parent = {},
       args: GeneralArgs = {},
       _context = {}
-    ): Observable<JobConnection | null> {
-      const {
-        sortBy = "ID",
-        sortDirection = "ASC",
-        filter = null,
-        namespace = null
-      } = args as JobsQueryArgs;
-
-      const pollingInterval$ = Observable.timer(0, pollingInterval);
-      const responses$ = pollingInterval$.exhaustMap(fetchJobs);
-
-      const jobsInNamespace$ = responses$.map(jobs =>
-        filterJobsByNamespace(jobs, namespace)
-      );
-      const count$ = jobsInNamespace$.map(jobs => jobs.length);
+    ): Observable<JobConnection> {
+      if (!isJobsQueryArg(args)) {
```

Comment:
I think this is a bit strange as the check only "by accident" that the path should be part of the argument. I would either like the error message to read something like "you need to pass in a JobsQuerArg" or have a test like `if (!args.path)`.

I want to prevent us from having the error check and message drift from each other.

Reply:
updated Errormessage below 👍